contrib/storage-jdbc/pom.xml

```diff
@@ -15,62 +15,241 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <artifactId>drill-contrib-parent</artifactId>
-    <groupId>org.apache.drill.contrib</groupId>
-    <version>1.2.0-SNAPSHOT</version>
-  </parent>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>drill-contrib-parent</artifactId>
+        <groupId>org.apache.drill.contrib</groupId>
+        <version>1.2.0-SNAPSHOT</version>
+    </parent>
 
-  <artifactId>drill-jdbc-storage</artifactId>
+    <artifactId>drill-jdbc-storage</artifactId>
 
-  <name>contrib/jdbc-storage-plugin</name>
+    <name>contrib/jdbc-storage-plugin</name>
 
-  <dependencies>
+    <properties>
+        <mysql.connector.version>5.1.36</mysql.connector.version>
+        <derby.reserved.port>20000</derby.reserved.port>
```

Comment:
I know that @julienledem was recently trying to make Drill's test use port hunting to avoid conflicts between two separate test suites running simultaneously. I'm assuming that isn't possible here but I thought I should check.